exasol/analytics/query_handler/udf/runner/udf.py

```diff
@@ -187,10 +211,10 @@ def release_and_create_query_handler_context_if_input_query(
             current_state.top_level_query_handler_context.get_child_query_handler_context()
         )
 
-    def _get_parameter(self, ctx):
+    def _get_parameter(self, ctx) -> UDFParameter:
         iter_num = ctx[0]
```

Comment:
in another PR, we probably should replace the numbers with named constants

Reply:
See #228